resources/qml/TableView.qml

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2021 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
```

Comment:
Latest supported QtQuick for Qt5.10 is 2.7

Reply:
Due to conflicting documentation on the Qt website, we will leave it as is and see how our build system handles this version